components/date-picker/month-picker.component.spec.ts

```diff
@@ -146,7 +146,7 @@ describe('NzMonthPickerComponent', () => {
     it('should support nzClassName', () => {
       const className = fixtureInstance.nzClassName = 'my-test-class';
       fixture.detectChanges();
-      const picker = debugElement.query(By.css('.ant-calendar-picker')).nativeElement as HTMLElement;
+      const picker = debugElement.queryAll(By.css('.ant-calendar-picker'))[1].nativeElement as HTMLElement;
```

Comment:
it seems it has nothing todo with the pr with this code.

Reply:
@vthinkxie It's hard to test styles via javascript, see this [stackblitz](https://stackblitz.com/edit/ng-zorro-antd-start-yp2rts)

Reply:
@hungtcs the test is must for ci, you can use https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle to test the style in javascript.

Reply:
@vthinkxie Thank you, I will add the test case later.

Reply:
👍

Reply:
@vthinkxie Test case is completed.